src/pydata_sphinx_theme/theme/pydata_sphinx_theme/sections/announcement.html

```diff
@@ -1,2 +1,25 @@
-{# A banner with an alert. #}
-{# Note that this hasn't been optimized yet so may not behave as expected. #}
+{%- if theme_announcement -%}
+
+{% set is_remote=theme_announcement.startswith("http") %}
+
+{# If we are remote, add a script to make an HTTP request for the value on page load #}
+{%- if is_remote %}
+<script>
+$.get("{{ theme_announcement }}", success=(data)=>{
+      console.log(data);
```

Comment:
question: is this considered JS best practice (to `console.log` on successful `get`)? Or is this cruft left over from the process of developing this feature? (I honestly don't know, but my instinct would be to only log the failures)

Reply:
it is cruft!

Reply:
if that's the case I need to edit the theme code as well I think I'm `consol.log` every time the mode is changed